docs/guides/formulasC2/referenceAPI.md

```diff
@@ -0,0 +1,531 @@
+      "properties": {
+        "method": "POST",
+        "elementInstanceId": "${instanceName2}",
+        "api": "/hubs/messaging/messages",
+        "body": "${steps.step2}"
+      }
+    }
+  ]
+}
+
+
+```
+
+## Triggers
+
+Triggers are the actions that kick off a formula. Triggers can be one of the following type:
+
+* [Event](#event)
+* [Element Request](#element-request)
+* [Scheduled](#scheduled)
+* [Manual](#manual)
+
+### Event
+
+You can set up triggers that listen for an event to happen on an element instance. To set up this trigger, you must use an Element Instance Variable that, when specified in a formula instance, refers to an element instance that is configured to use webhooks or polling to listen for events.
+
+To set up an Event trigger:
+
+* Specify the `type` as `event`.
+* For  `elementInstanceId` include the Element Instance Variable that triggers the formula.
+
+```json
+"triggers": [{
+  "type": "event",
+  "properties": {
+    "elementInstanceId": "${crmElement}"
+    },
+  "onSuccess": ["step1"]
+}]
+```
+
+If an Event trigger's Element Instance is set up for polling instead of webhooks, then each object that is found while polling triggers a separate formula execution. For example, if the poller finds five changes, five different formula executions kick off.
+
+To see event triggers in action, see the following examples:
+
+* [CRM to Messages](examples.html)
+* [Retrieve, Transform, and Sync Contact](examples.html#retrieve-transform-and-sync-contact)
+
+{% include Formulas/scope-trigger-event.md %}
+
+### Element Request
+
+Triggered any time a specific API call is made to a given Element Instance. To set up this trigger, you must use an Element Instance Variable that, when specified in a formula instance, refers to an element instance.
+
+To set up an Element Request trigger:
+
+* Specify the `type` as `elementRequest`.
+* In `properties`:
+  * For `elementInstanceId`, include the Element Instance Variable that triggers the formula.
+  * For `method`, specify a valid API verb. {{site.data.table-desc.formula-method}}
+  * For `api` enter the endpoint, such as `hubs/crm/contacts`
+
+```json
+"triggers": [{
+  "type": "elementRequest",
+  "properties": {
+    "method": "POST",
+    "elementInstanceId": "${config.crmInstance}",
+    "api": "/hubs/crm/contacts"
+    },
+  "onSuccess":"[step1]"
+}]
+  ```
+
+### Scheduled
+
+Triggered at times specified by a Cron job. We recommend that you review the many reference pages for Cron jobs online.
+
+{% include note.html content=" The minimum scheduled frequency is 15 minutes." %}
+
+In general, the Cron format consists of:
+
+| Minute </br>0-59| Hour</br>0-23   | Day of Month</br>1-31   | Month of Year</br>1-12   | Day of Week</br>1-7</br>Monday-Sunday   | Year</br>1900-3000   |
+| :------------- | :------------- | :------------- | :------------- | :------------- |:------------- |
+|  \*  |  \*  |  \*  |  \*  |  \*  |  \*  |
+
+#### Examples
+
+* Run each minute
```

Comment:
General point, but just putting it here: it seems this is redundant with a lot of the data we had previously about all of the different trigger/step types.  Can we put this in one place to make maintenance of it easier?